client/checkout/classic/index.js

```diff
@@ -416,9 +438,15 @@ jQuery( function ( $ ) {
 				)
 			);
 		}
-		if ( isWCPayGiropayChosen() ) {
-			// Giropay does not use saved payment methods at this time
-			return false;
+
+		if ( isWCPaySofortChosen() ) {
+			return (
```

Comment:
Will Sofort implement saving a payment method? I wasn't sure if it matched Giropay on this, but if it does, this bit could be skipped?

Reply:
[Stripe seems to indicate that Sofort can be used as a saved payment method](https://stripe.com/docs/payments/sofort/save-during-payment). However, unlike SEPA, there aren't really any fields to save at checkout. The only information I need to send to Stripe before the jump is the user's billing country, which is saved by the checkout already. 

Since I don't think this section of code is really doing anything at the moment, because there aren't any additional fields to enter when you select Sofort as a payment method, I've removed this section for now and created #1636 to look into this one properly.

Cheers for checking, @bborman22 ! Let me know if you think this has the green light now and I'll merge away.

Reply:
Yes, this looks good to me!

Reply:
Just did a quick test and it still worked as expected, so should be good either way!

Reply:
I just removed the whole thing thinking that since the payment method _can_ be saved, even though I'm not at the moment certain which fields need to be saved, I'd just let `isUsingSavedPaymentMethod` return the default result and resolve this fully in the separate issue.